src/reducers/src/provider-state-updaters.ts

```diff
@@ -382,72 +379,3 @@ export const resetProviderStatusUpdater = (state: ProviderState): ProviderState
   isCloudMapLoading: false,
   successInfo: {}
 });
-
-/**
- * Set current cloudProvider
- */
```

Comment:
we no longer need these actions since we are handling everything through hooks and provider